### DISCUSSION TITLE
Prometheus metrics explorer

### DISCUSSION THREAD

**Discussion by bohandley** [grafana/grafana#64574] [Data Sources]
Ask a question or provide feedback on the Metric Encyclopedia.

Want to sign up for a 20minute interview? Contact me or fill out this form and leave your email
[Metric Encyclopedia Google Form ](https://docs.google.com/forms/d/e/1FAIpQLSeMsdPrHfKOyynT-zlbFF22gtCfU_Fw5V7UU3ufZ34ZtZlbzg/viewform)

https://github.com/grafana/grafana/pull/63423

**Comment by leeoniya** [comment]
i found these label-less toggles to be [confusing](https://en.wikipedia.org/wiki/Mystery_meat_navigation):

![image](https://user-images.githubusercontent.com/43234/224187718-efa4f4ea-bc37-4c76-a05e-bc3cd760c2d2.png)

**Reply by bohandley** [reply]
Agreed! UX also agreed so they will be given labels

**Comment by bohandley** [comment]
Renamed to metrics explorer. Blog post coming soon!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as an ongoing discussion, and we'd like those to go to our[ community forum](https://community.grafana.com/).
